packages/generator-open-wc/generators/app/index.js

```diff
@@ -1,10 +1,54 @@
+/* eslint-disable no-console */
+
 const Generator = require('yeoman-generator');
+const inquirer = require('inquirer');
+
+const choices = {
+  appStarter: 'Create Open Webcomponents Starter App',
```

Comment:
```suggestion
  appStarter: 'Create Open Web Components Starter App',
```

Reply:
I can't apply these sugguestions? (maybe only create of MR can do it?)

can you? if so pls do... if not I could do it "manually"